pmm-app/src/shared/components/Elements/Icons/Database.tsx

```diff
@@ -0,0 +1,55 @@
+/* eslint max-len: 0 */
+
+import React, { FC } from 'react';
+import { useTheme } from '@grafana/ui';
+import { getStyles } from './Icons.styles';
+
+export const Database: FC = (props) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <svg
```

Comment:
Since all icons look the the same on the screen, can we put the svg to a separate file under `/assets`?

Reply:
Makes sense, I'll create separate task for Icons improvements

Reply:
I'd like to get it fixed now - we should store icons in svg files, not in code.

Reply:
While a separate task would fix the problem elsewhere and some time later, I would like to fix what's in the scope of this ticket.